### DISCUSSION TITLE
IMPORTANT ENDPOINT

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6490] [General]
in ENDPOINT we have only RTMP , please add SRT so will be able to send to SRT  destination .

**Comment by brixxnatt** [comment]
anyone ?

**Reply by asharamseervi** [reply]
Hi @brixxnatt,

_Could I kindly ask you to make your thread titles concise and your questions clear, with a brief description about your objectives and use-cases in the body? This will help the community and team members to better understand and respond to your queries more effectively. Now, coming to your question based on what I understood is:_

As per @USAMAWIZARD previous comment here: https://github.com/orgs/ant-media/discussions/6041#discussioncomment-8177771
you may use SRT and RTMP with the same Input field at the Ant Media Management Console. Correct me if I'm wrong @USAMAWIZARD.

Kindly try once publishing SRT through the same RTMP field; despite there is only mentioned about RTMP URL; it should work.

For your further reference on SRT publishing, you may follow the official documentation here: [https://antmedia.io/docs/guides/publish-live-stream/srt/](https://antmedia.io/docs/guides/publish-live-stream/srt/?utm_source=github&utm_medium=discussions) and Blog Post: [https://antmedia.io/how-to-ingest-srt-and-play-with-webrtc/](https://antmedia.io/how-to-ingest-srt-and-play-with-webrtc/?utm_source=github&utm_medium=discussions)

**Comment by brixxnatt** [comment]
I could not push SRT in the endpoint , thats why I posted the question , and about the title , what would you like the title be please .

**Reply by brixxnatt** [reply]
and as i can see you are in AMS team , did you try to push SRT in the endpoint ?

**Reply by yashtandon113** [reply]
HI @brixxnatt 
Currently, Ant Media Server does not support re-streaming to SRT ingest points. We also have a feature request on our side: https://github.com/ant-media/Ant-Media-Server/issues/4450.

It is currently in the backlog, and there is no ETA for scheduling this issue. If this feature is very important to you, please contact contact@antmedia.io so that we can discuss it further.
